api/boilerplate/views.py

```diff
@@ -1,7 +1,47 @@
+from uuid import UUID
 from django.contrib.auth.models import User, Group
 from rest_framework import viewsets
 from .serializers import UserSerializer, GroupSerializer
+from django.shortcuts import redirect, render
+from django.http import HttpResponseBadRequest
+from eid_service.models import AuthenticationRequest
 
+def get_tc_token_url(request):
+    if request.method == 'GET':
+        protocol = request.GET.get('protocol', 'http')
+        host = request.GET.get('host', '127.0.0.1')
+        if (protocol != 'http') and (protocol != 'eid'):
+            return HttpResponseBadRequest()
+        if (host != '127.0.0.1') and (host != 'localhost'):
+            return HttpResponseBadRequest()
+        authenticationRequest = AuthenticationRequest()
+        authenticationRequest.save()
+        tc_token_url = 'https://' + request.get_host() + '/api/getTcToken?tcTokenId=' + str(authenticationRequest.tcTokenId)
+        return redirect(protocol + '://' + host + ':24727/eID-Client?tcTokenUrl=' + tc_token_url)
+    else:
+        return HttpResponseBadRequest()
+
+def get_tc_token(request):
+    if request.method == 'GET':
+        try:
+            tcTokenId = UUID(request.GET.get('tcTokenId'))
+        except ValueError:
+            return HttpResponseBadRequest()
+        except TypeError:
+            return HttpResponseBadRequest()
+        try:
+            authenticationRequest = AuthenticationRequest.objects.get(tcTokenId=tcTokenId)
+        except AuthenticationRequest.DoesNotExist:
+            return HttpResponseBadRequest()
+        context = {
+            'serverAddress': 'https://...',
```

Comment:
This is a stub and must be filled by following pull requests implementing the eID-Server.